action.yml

```diff
@@ -74,7 +74,11 @@ runs:
       id: determine_next_tag_retrospectively
       run: |
         echo "Determining whether user selected an auto-tag option..."
-        script_output=$("${{ github.action_path }}/auto-tag.sh" "${{ fromJson(steps.get_pr_data.outputs.result).body }}" "${{ fromJson(steps.get_pr_data.outputs.result).number }}")
+
+        export PR_BODY=${{ fromJson(steps.get_pr_data.outputs.result).body }}
+        export ESCAPED_PR_BODY="$(echo $PR_BODY | envsubst)"
```

Comment:
Am I missing something - I can't see any env vars in the PR Body?

Reply:
there aren't... let's see if removing the double quotes here works though, as this is pretty urgent!